### PR TITLE
Michielkallenberg/evaluation

### PR DESCRIPTION
**Pull request by michielkallenberg**
Added some evaluation code. 
The test_model.py now also does the evaluation; results are hardcoded, as "Ridge" seems deterministic.  Not sure whether this is to your liking. Just let me know.

### CODE REVIEW

**Review by krsnapaudel — CHANGES_REQUESTED** [review]
I think MAPE is incorrect. The division must be done for each sample:
```
# This definition is from the suggested answer to:
# https://stats.stackexchange.com/questions/58391/mean-absolute-percentage-error-mape-in-scikit-learn/294069#294069
def mean_absolute_percentage_error(Y_true, Y_pred):
  """Mean Absolute Percentage Error"""
  Y_true, Y_pred = np.array(Y_true), np.array(Y_pred)
  return np.mean(np.abs((Y_true - Y_pred) / Y_true)) * 100
```

**Comment by ritviksahajpal** [comment]
@michielkallenberg and @krsnapaudel  Scikit-learn has MAPE: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html, why not just use that rather than reinventing the wheel? I realize it is a simple function, but it makes sense to me to use an established library where available unless your function is adding something that the library does not have

**Comment by michielkallenberg** [comment]
Hi @ritviksahajpal that's what I just did this morning :-)

**Comment by ritviksahajpal** [comment]
@michielkallenberg my bad! I should have checked the updated code first. I think as long as you document it , it is fine. One possible addition could be to remove NaNs in the y_true and y_pred if any exist. I find that when running yield models in real world scenarios, it is helpful to either flag or filter out NaNs. sklearn MAPE will likely crash if it encounters NaNs so any NaN handling will either need to be flagged via exception or handled outside of it

**Comment by michielkallenberg** [comment]
No worries at all @ritviksahajpal. Thank you for your contribution. I've made a brief mention of the multiplication-by-100 issue. It has also been noted disapprovingly by others, as indicated here: https://github.com/scikit-learn/scikit-learn/issues/23408.
Regarding NaN values, scikit-learn will generate an error upon encountering them. I believe it's acceptable to delegate the NaN check to scikit-learn. However, if someone were to implement their own metrics, this check might be overlooked.

**Comment by krsnapaudel** [comment]
@ritviksahajpal, @michielkallenberg : I had not checked the most recent scikit metrics. I guess this was added since I developed `cypml`. Yes let's use their metric.

**Review by krsnapaudel — APPROVED** [review]
I think the deterministic behavior of Ridge is expected. By default, they use a seed.
